### PR TITLE
Updating how the script tags are used in the index.html files.

### PR DESCRIPTION
**Pull request by Sfshaza**
@kwalrath

### CODE REVIEW

**Comment by kwalrath** [comment]
It looks like you might have missed updating streams/read_file/pubspec.yaml. Continuing to look...

File: futures/pubspec.yaml [hunk 1]
```diff
@@ -1,2 +1,6 @@
 name: futures
 description: A sample application
+
+homepage: https://www.dartlang.org/docs/tutorials/futures/
+gist: https://gist.github.com/Sfshaza/9e43fdcdc8848cd1490b
+dartpad: https://dartpad.dartlang.org/9e43fdcdc8848cd1490b
```
Comment (kwalrath) [inline futures/pubspec.yaml]: This is a dart:io sample, so kill the dartpad link. I guess it's OK to keep the gist link?

Maybe we should file an issue on generator-gist asking not to create the dartpad link if the file relies on dart:io or packages?

File: streams/throw_error/pubspec.yaml [hunk 2]
```diff
@@ -3,4 +3,6 @@ description: A simple app that reads from a stream and throws an error when the
 dependencies:
   args: ^0.10.0
 
-homepage: http://gist.github.com/Sfshaza/d68171ddfdd8edeab932
\ No newline at end of file
+homepage: http://gist.github.com/Sfshaza/d68171ddfdd8edeab932
```
Comment (kwalrath) [inline streams/throw_error/pubspec.yaml]: This should be https://www.dartlang.org/docs/tutorials/streams/

File: streams/simple_stream/pubspec.yaml [hunk 3]
```diff
@@ -3,4 +3,6 @@ description: A simple application that reads integers from a stream and adds the
 dependencies:
   args: ^0.10.0
 
-homepage: http://gist.github.com/Sfshaza/6500c61e0fb3ed3ed612
\ No newline at end of file
+homepage: http://gist.github.com/Sfshaza/6500c61e0fb3ed3ed612
```
Comment (kwalrath) [inline streams/simple_stream/pubspec.yaml]: This should be https://www.dartlang.org/docs/tutorials/streams/

File: streams/last_positive/pubspec.yaml [hunk 4]
```diff
@@ -3,4 +3,6 @@ description: A simple app that reads from an integer stream and reports the last
 dependencies:
   args: ^0.10.0
 
-homepage: http://gist.github.com/Sfshaza/28f22ba28e2360b77f40
\ No newline at end of file
+homepage: http://gist.github.com/Sfshaza/28f22ba28e2360b77f40
```
Comment (kwalrath) [inline streams/last_positive/pubspec.yaml]: This should be https://www.dartlang.org/docs/tutorials/streams/

**Comment by kwalrath** [comment]
lgtm, once you fix the homepage links, update streams/read_file/pubspec.yaml, and whatever else I said...

File: futures/pubspec.yaml (hunk 1 above)
Comment (Sfshaza) [inline futures/pubspec.yaml]: I filed a bug: https://github.com/kasperpeulen/gist-generator/issues/10

**Pull request merged**